src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/SymbolType.cs

```diff
@@ -14,8 +16,20 @@ internal enum TypeKind
     }
 
     // This is a kind of Type object that will represent the compound expression of a parameter type or field type.
-    internal sealed class SymbolType : TypeInfo
+    internal sealed partial class SymbolType : TypeInfo
     {
+        #region Data Members
+        internal Type _baseType = null!;
```

Comment:
This needs a comment which states that mono is directly reading these fields, so the order etc. matters.